pyproject.toml

```
@@ -30,10 +30,15 @@ classifiers = [
 dynamic = ["version", "readme"]
 
 dependencies = [
-    "botocore >=1.35.74, <1.35.89", # NOTE: When updating, always keep `project.optional-dependencies` aligned
     "aiohttp >=3.9.2, <4.0.0",
-    "wrapt >=1.10.10, <2.0.0",
     "aioitertools >=0.5.1, <1.0.0",
+    "botocore >=1.35.74, <1.35.89", # NOTE: When updating, always keep `project.optional-dependencies` aligned
+    "python-dateutil >=2.1, <3.0.0",
+    "jmespath >=0.7.1, <2.0.0",
+    "multidict >=6.0.0, <7.0.0",
+    "urllib3 >=1.25.4, <1.27; python_version<'3.10'",
+    "urllib3 >=1.25.4, !=2.2.0, <3; python_version>='3.10'",
```

Style nit: I think that a sparser style is easier to read:
```suggestion
    "urllib3 >= 1.25.4, < 1.27; python_version < '3.10'",
    "urllib3 >= 1.25.4, != 2.2.0, <3; python_version >= '3.10'",
```